app/react/theme/index.ts

```diff
@@ -0,0 +1,66 @@
+const theme = {
+	colors: {
+		primary: {
+			500: "#3d9970",
+		},
+		secondary: {
+			500: "#dd7c58",
+		},
+		gray: {
+			100: "#f8f8f8",
+		},
+		white: "#fff",
+		black: "#000",
+	},
+	shadows: {
+		default: "0 0 20px rgb(0, 0, 0, 0.1)",
+	},
+	radii: {
```

Comment:
Radii? I didn't understand this part 😞

Reply:
Is to define `border-radius` in our theme. We'll start the migration with the Login page, so I already defined it to use in the `Card` component. But it is all open to discussion, mainly this theme definition.